integration-tests/tests/src/tests/queries.ts

```diff
@@ -126,7 +126,7 @@ const expectQueryResultValues = (
     expect(expectedResults).to.deep.equal(
```

Comment:
The PR is titled "session tests" - were these changes to the queries tests intentional?